### DISCUSSION TITLE
Can I use record.Get for relation?

### DISCUSSION THREAD

**Discussion by baoboine** [pocketbase/pocketbase#2554] [Q&A] [answered]
I'm using Pocketbase as Framework. I have a question, can I get relation value with `record.Get` ? 

For example: `record.Get("user.email")` or `record.Get("user.id")`

Thank you

**Comment by ganigeorgiev** [comment]
No. `record.Get()` returns only the plain value, eg. the relation id.

If you want to get a relation, you'll need to fetch the relation separately, aka.:

```go
app.Dao().FindRecordById("example", record.GetString("yourRelationField"))
```

**Reply by baoboine** [reply]
@ganigeorgiev so how can I get collectionId with RelationField ?

**Reply by ganigeorgiev** [reply, marked as answer]
The collection id associated with the `relation` field is part of the collection's schema field options. You can access it by doing something like:

```go
field := record.Collection().Schema.GetFieldByName("yourRelationField")
options := field.Options.(*schema.RelationOptions)
log.Println(options.CollectionId)
```

**Reply by baoboine** [reply]
@ganigeorgiev Thanks for your help. I love you so much. If I ever get the chance to visit Bulgaria, I'll be sure to look you up! :D
